apps/pcq/pcq-backend-int/demo.yaml

```diff
@@ -16,9 +16,9 @@ spec:
         pcq:
           resourceGroup: pcq
           secrets:
-            - name: backend-POSTGRES-USER-FLEXIBLE
+            - name: backend-POSTGRES-USER
               alias: PCQ_DB_USERNAME
-            - name: backend-POSTGRES-PASS-FLEXIBLE
+            - name: backend-POSTGRES-PASS
```

Comment:
are PCQ using -int?

Reply:
saw issue is not the same on kv secret as it cannot pull it

Reply:
no other reference to match that secret so it looks like it is the correct one